bazel/envoy_internal.bzl

```diff
@@ -129,9 +129,14 @@ def envoy_copts(repository, test = False):
            envoy_select_signal_trace(["-DENVOY_HANDLE_SIGNALS"], repository) + \
            _envoy_select_path_normalization_by_default(["-DENVOY_NORMALIZE_PATH_BY_DEFAULT"], repository)
 
+_DEPS = {
+    "grpc_transcoding": "@grpc_httpjson_transcoding//src:transcoding",
+    "path_matcher": "@grpc_httpjson_transcoding//src:path_matcher",
+}
+
 # References to Envoy external dependencies should be wrapped with this function.
 def envoy_external_dep_path(dep):
-    return "//external:%s" % dep
+    return _DEPS.get(dep, "//external:%s" % dep)
```

Comment:
If it's as clean as this in general, I have no objection.

Reply:
Once everything is migrated we can just do `_DEPS[dep]` instead

Reply:
could we make the constant something `_BINDS` just because i have had (and noticed) confusion about how binds works - and they are targets rather than repositories - so im thinking that would be clearer

Reply:
yep, done